packages/@uppy/companion/package.json

```diff
@@ -52,6 +52,7 @@
     "jsonwebtoken": "8.5.1",
     "lodash": "^4.17.20",
     "mime-types": "2.1.25",
+    "moment": "^2.29.1",
```

Comment:
That doesn't seem to be necessary.

Reply:
I think `moment-timezone` used to include it but because of an update you now need to install it separately?